react/features/toolbox/components/AbstractToolbarButton.js

```diff
@@ -50,7 +50,7 @@ export type Props = {
  *
  * @abstract
  */
-export default class AbstractToolbarButton<P: Props> extends Component<P> {
+export default class AbstractToolbarButton<P: Props, State=void> extends Component<P, State> {
     /**
```

Comment:
do we need to explicitly define state as void here?

Reply:
Yes, otherwise we need to pass void for any component that extends it